controllers/clusterresources/redisuser_controller.go

```diff
@@ -267,11 +267,11 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	if user.DeletionTimestamp != nil {
 		err = r.handleDeleteUser(ctx, l, secret, user)
 		if err != nil {
-			return models.ReconcileRequeue, nil
+			return ctrl.Result{}, err
 		}
 	}
 
-	return models.ExitReconcile, nil
+	return ctrl.Result{}, nil
```

Comment:
CustomRateLimiter is not added to the Redis User controller

Reply:
Fixed